core/src/main/java/c/org/rajawali3d/control/RenderContextType.java

```diff
@@ -0,0 +1,16 @@
+package c.org.rajawali3d.control;
+
+/**
+ * Basic render context types
+ *
+ * @author Randy Picolet
+ */
+public enum RenderContextType {
```

Comment:
Can we make this an android `@IntDef annotation instead?

Reply:
Makes sense, sure.